web/frontend/src/lib/components/robot-client.svelte

```diff
@@ -75,7 +76,13 @@ $robotClient = new Client(impliedURL, {
   noReconnect: true,
 });
 
-let password = '';
+const passwordByAuthType: {[key in string]: string} = {}
+const authEntityByAuthType: {[key in string]: string} = {}
```

Comment:
Since entity is only used for the "api-key" auth type, I would simplify this to be just a single value:

```
let apiKeyEntity = "";
```